controllers/watcherapplier_controller.go

```diff
@@ -231,6 +251,7 @@ func (r *WatcherApplierReconciler) initStatus(instance *watcherv1beta1.WatcherAp
 		condition.UnknownCondition(condition.InputReadyCondition, condition.InitReason, condition.InputReadyInitMessage),
 		condition.UnknownCondition(condition.ServiceConfigReadyCondition, condition.InitReason, condition.ServiceConfigReadyInitMessage),
 		condition.UnknownCondition(condition.MemcachedReadyCondition, condition.InitReason, condition.MemcachedReadyInitMessage),
+		condition.UnknownCondition(condition.DeploymentReadyCondition, condition.InitReason, condition.DeploymentReadyMessage),
```

Comment:
s/DeploymentReadyMessage/DeploymentReadyInitMessage

Reply:
Good catch, fixed in the lastest commit, thanks!